scripts/workers/code_upload_worker_utils/deploy_calico.sh

```diff
@@ -31,6 +31,14 @@ echo "### Kubectl Installed"
 curl https://raw.githubusercontent.com/aws-samples/amazon-cloudwatch-container-insights/latest/k8s-deployment-manifest-templates/deployment-mode/daemonset/container-insights-monitoring/quickstart/cwagent-fluentd-quickstart.yaml | sed "s/{{cluster_name}}/$CLUSTER_NAME/;s/{{region_name}}/$AWS_DEFAULT_REGION/" | kubectl apply -f -
 echo "### Container Insights Installed"
 
+# Install cilium
```

Comment:
Can we please change the name of this file to something meaningful like -- `install_dependencies.sh`. The current name is misleading.